flipt-engine/src/models/common.rs

```diff
@@ -18,6 +27,15 @@ pub enum SegmentOperator {
     And,
 }
 
+impl From<i32> for SegmentOperator {
+    fn from(n: i32) -> SegmentOperator {
+        match n {
+            1 => SegmentOperator::And,
+            _ => SegmentOperator::Or,
```

Comment:
```suggestion
            0 => SegmentOperator::Or,
            _ => SegmentOperator::Or,
```

Same here, maybe we should be all the way explicit.